### DISCUSSION TITLE
Where are the logs?

### DISCUSSION THREAD

**Discussion by BarryBahrami** [drakkan/sftpgo#1655] [Q&A]
Hello,

I am just checking out SFTPGo.  So far looks pretty good.  But I'm not sure where the logs are being saved?  And can they be pointed to an Azure blob storage account?  how?

thank you much

**Comment by DHolmes01** [comment]
It depends which machine you are on.
Linux: `/etc/sftpgo/sftpgo.log`
Windows: `C:\ProgramData\SFTPGo\logs`
I am not 100% sure about Mac
